templates/zerver/integrations.html

```diff
@@ -1153,6 +1159,25 @@
 
     </div>
 
+    <div id="helloworld" class="integration-instructions">
+
+        <p>Learn how Zulip integrations work with this simple Hello World example!</p>
+
+        <p>The Hello World webhook will use the <code>test</code> stream, which is
+        created by default in the Zulip dev environment. If you are running
+        Zulip in production, you should make sure this stream exists.</p>
+
+        <p>Next, on your <a href="/#settings" target="_blank">Zulip
+        settings page</a>, create a Hello World bot.  Construct the URL for
+        the Hello World bot using the API key and stream name:
+          <code>{{ external_api_uri }}/v1/external/helloworld?api_key=abcdefgh&amp;stream=test</code>
+        </p>
+
+        <p><b>Congratulations! You're done!</b><br /> Your messages may look like:</p>
+
+        <img class="screenshot" src="/static/images/integrations/helloworld/001.png" />
+    </div>
```

Comment:
For this integration in particular, we should probably document how to trigger a notification, since HelloWorld isn't a real product as far as I know :).